### ISSUE TITLE
Outdated dependencies + native library linking issue

### ISSUE THREAD

**Issue by ethanshry** [jgallagher/amiquip#20]
I'm working with amiquip in a project and would like to be able to use an up-to-date version of the git2 crate (0.13.5), however I am running into outdated dependency issues with the version of built that amiquip depends on:

```
multiple packages link to native library git2, but a native library can be linked only once

package libgit2-sys v0.7.1
    ... which is depended on by git2 v0.7.1
    ... which is depended on by built v0.3.0
    ... which is depended on by amiquip v0.3.3
    ... [my project]
links to native library git2

package libgit2-sys v0.12.5+1.0.0
    ... which is depended on by git2 v0.13.5
    ... [my project]
also links to native library git2
```

I believe updating `amiquip's` dependency of `built` to 0.4.x will resolve this issue- are there any plans to update dependency versions for this crate any time soon?

I am also a rust newbie, so maybe there's a workaround I am not aware of?

**Comment by jgallagher** [comment]
I took a stab at updating all the dependencies a couple of weeks ago and bumped into a wall trying to upgrade `mio` (cf. dimbleby/mio-extras#7 and  tokio-rs/mio#1289). Since that's stalled it'd make sense to bump up all the ones that are more straightforward, though. If you want to do that (for just `built` or for any of the others) I'd happily take a look at a PR, or I'll try to get to it myself soon.

**Comment by jgallagher** [comment]
Thanks to @HonzaMatosik `built` is now upgraded. I'm going to close this for now, but feel free to reopen (or open a new issue) if building with 0.4.0 doesn't work.
